app/models/item.rb

```diff
@@ -0,0 +1,5 @@
+class Item < ApplicationRecord
+  has_one :list
```

Comment:
Why did you choose `has_one` instead of `belongs_to`?

Reply:
I thought the other class had the foreign key, I've changed it to `belongs_to` now.